src/overlays/actors/ovl_En_Ani/z_en_ani.h

```diff
@@ -6,9 +6,17 @@
 
 typedef struct {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x168];
+    /* 0x014C */ ColliderCylinderMain collider;
+    /* 0x0198 */ SkelAnime skelAnime;
+    /* 0x01DC */ Vec3s actorDrawTable[16];
+    /* 0x023C */ Vec3s transitionDrawTable[16];
+    /* 0x029C */ Vec3s unk_29C;
+    /* 0x02A2 */ Vec3s unk_2A2;
+    /* 0x02A8 */ u16 unk_2A8;
+    /* 0x02AA */ u16 unk_2AA;
+    /* 0x02AC */ s16 unk_2AC;
+    /* 0x02AE */ s16 unk_2AE;
+    /* 0x02B0 */ ActorFunc actionFunc;
 } EnAni; // size = 0x02B4
 
-extern const ActorInit En_Ani_InitVars;
```

Comment:
This is meant to stay in this file. Every actor should have its init vars exposed in their header.